ci/requirements-py34.yml

```diff
@@ -17,5 +17,7 @@ dependencies:
     - nose
     - pip:
         - coveralls
+        - pvfactors==1.0.1
```

Comment:
@anomam Do you know if I'm fixing or breaking something here? (Is pvfactors 1.0.1 really not supported in Python 3.4?)

Reply:
That's a good point @markcampanelli , starting with version 1.0.0 pvfactors should work with earlier versions of pandas, and therefore be compatible with Python 3.4.
Could you post an issue in pvfactors by any chance? A PR would be very much welcome as well :)

Reply:
Python 3.4 hit "end of life" a few months ago. I don't plan to support it going forward.

Reply:
https://github.com/SunPower/pvfactors/issues/49